### DISCUSSION TITLE
docs/self-hosted/search

### DISCUSSION THREAD

**Discussion by giscus** [flowershow/flowershow#835] [comments]
# docs/self-hosted/search

There are two kinds of providers to choose from when setting up search functionality on your flowershow site. These are kbar and algolia. The difference between the two is that algolia search allows f...

https://flowershow.app/docs/self-hosted/search

**Comment by yuri-karpovich** [comment]
Can Obsidian Canvas drawings be displayed? Do you have it on your roadmap? It would be a game-changing feature that could replace Miro for me

**Reply by rufuspollock** [reply]
@yuri-karpovich thanks for asking. We don't support that at present. It is a nice idea and ad I've added to the roadmap backlog.

**Comment by jjfantini** [comment]
Do we use Search or Wirte API key?

**Comment by olayway** [comment]
Closing this issue since it concerns the old, now-deprecated self-hosted Flowershow Next.js template.
